src/NzbDrone.Core/NetImport/TMDb/TMDbSettings.cs

```diff
@@ -64,6 +64,7 @@ public TMDbSettings()
             MinVoteAverage = "5";
             MinVotes = "1";
             LanguageCode = (int)TMDbLanguageCodes.en;
+            AccountID = "https://github.com/elChapoSing/tmdb_account_id";
```

Comment:
Can you move this link to the FieldDefinition? There is a HelpLink property on that, which will automatically create a hyperlink.